mailpoet/lib/API/JSON/v1/Settings.php

```diff
@@ -493,4 +495,25 @@ public function reactivateReEngagementEmails(): array {
       'action' => 'reactivate',
     ];
   }
+
+  /**
+   * Prepares the settings to set up MSS with the given key and calls the set method.
+   *
+   * @param string $apiKey
+   * @return ErrorResponse|SuccessResponse
+   */
+  public function setupMSS(string $apiKey) {
```

Comment:
Now that the method saves also the premium key maybe we could use a different name. Maybe something like `setKeyAndSetupMss` ? Hmm I'm not sure maybe I'm overthinking this a bit. If you think it is fine you can ignore this comment.